### ISSUE TITLE
Issues with contrast on the site

### ISSUE THREAD

**Issue by deakjahn** [dart-lang/site-www#1710]
But really? Who thought that this would be legible?

**Comment by sfshaza2** [comment]
What? Where? I gather the contrast of something is insufficient..

**Comment by deakjahn** [comment]
The site. Am I in the wrong place? I assumed it was about the web site eg. https://dart.dev/guides/language/language-tour

Main text area is so-so, border case, I'd use a more definite color but can be disputed. Code samples are downright pale, especially where the background matches the foreground (red on red, gray on gray). Right side menu is abysmal, both by eye and by measurement.

What I really don't understand is the basic rationale. I know this is the trend in fashion and lifestyle websites but I don't think a reference material for a programming language should follow fashion trends. Simply two different worlds—fortunately, I'd say :-)

**Comment by sfshaza2** [comment]
Thanks for that clarification, @deakjahn.

**Comment by johnpryan** [comment]
I made some changes [here ](https://github.com/dart-lang/site-www/pull/1574)a while back that might improve this.

**Comment by sfshaza2** [comment]
Good point, @johnpryan. We should revisit this, or maybe run accessibility tools over some of the pages.
